src/rulesets/resource-names.ts

```diff
@@ -0,0 +1,17 @@
+const pluralize = require("pluralize");
+
+// Here's your escape hatch to customize plural rules for your domain
+// https://www.npmjs.com/package/pluralize#usage
+
+// pluralize.addUncountableRule("snyk"); // no way to make this plural
+// pluralize.addPluralRule("bacterium", "bacteria");
```

Comment:
here is where we can teach it about weird words that may exist in the Snyk domain that don't fit nicely